### PR TITLE
Remove hover on workspace page items

### PR DESCRIPTION
**Pull request by shafi1379**
@afoerster

### CODE REVIEW

File: ui/index.less [hunk 1]
```diff
@@ -33,6 +33,7 @@ tbody {
   font-weight: 300 !important;
   padding: 8px 0 4px 0 !important;
   margin: 0 32px !important;
+  cursor: default !important;
```
Comment (afoerster) [inline ui/index.less:36]: @shafi1379 this is set at the body level, so does this affect any other hovers?

Comment (rk-fsdev) [reply ui/index.less:36]: @shafi1379 , I agree with @afoerster 's opinion. I think it maybe affect any other hovers.

Comment (shafi1379) [reply ui/index.less:36]: @afoerster @rk-fsdev Added specifically for `.ant-tabs-tab` class not for tbody. It seems like that but please expand the file and see (By clicking on that line `@@ -33,6 +33,7 @@ tbody {`).

Comment (shafi1379) [reply ui/index.less:36]: ARCH-687

**Review by afoerster — APPROVED** [review]

**Pull request merged**